client/scss/tools/_mixins.guide-line.scss

```diff
@@ -49,4 +66,14 @@ $guide-line-default-color: theme('colors.grey.150');
       --guide-line-color: Highlight;
     }
   }
+
+  // For browsers supporting :has, only use the "highlight" guidelines
+  // for child elements in hover/focus and their descendants.
+  &:has(.w-panel:is(:hover, :focus-within)) {
```

Comment:
I thought this would be an appropriate use of `:has`. The alternative would be a lot of descendant selectors with overrides, possible but a _lot_ of very ugly CSS just to set a color.